Sources/Soto/Services/EC2/EC2_Shapes.swift

```diff
@@ -336,6 +336,7 @@ extension EC2 {
         case cancelled
         case cancelling
         case completed
+        case deleted = "DELETED"
```

Comment:
Why is this the only one that's uppercased?

Reply:
Looks like my patch was wrong. The EC2 models don't have the `deleted` state in them, but the EC2 waiter models depend on this state being there. I patched the EC2 models, but looking at the waiter I did it wrong.